### PR TITLE
Made changes specific to Charlotte Azure Event

### PR DESCRIPTION
**Pull request by nirajrules**

### CODE REVIEW

File: 2023/CharlotteMicrosoftAzure/data.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "GitHubHandles": [
+      "nirajrules",
+      "ashishmgupta"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Charlotte Microsoft Azure",
+    "Country": "United States",
+    "Url": "https://www.meetup.com/charlotte-microsoft-azure1/",
+    "SessionizeApiId": "{Your sessionize event Id}",
```
Comment (noopman) [inline 2023/CharlotteMicrosoftAzure/data.json:13]: Are you planning to use Sessionize to show off your community's sessions on the global azure site?

**Review by noopman — CHANGES_REQUESTED** [review]
I am missing an index.md file in this submission, please, @nirajrules @ashishmgupta.

**Comment by noopman** [comment]
*Ping* @nirajrules @ashishmgupta !

**Comment by tiagocostapt** [comment]
@nirajrules do you still want to move this PR forward?

**Comment by noopman** [comment]
I pinged @ashishmgupta on LinkedIn.

**Comment by noopman** [comment]
Closing this due to inactivity!

**Pull request closed without merging**